WordPressKitTests/Models/Stats/V2/Insights/StatsDotComFollowersInsightTests.swift

```diff
@@ -0,0 +1,129 @@
+import XCTest
+@testable import WordPressKit
+
+class StatsDotComFollowersInsightTests: XCTestCase {
+
+    private enum Constants {
```

Comment:
In tests, my main priority is always legibility, so while I don't see this extraction as a problem I think we can keep this string where it's needed?

Reply:
Fair point. I was thinking from the angle of "I don't want to scroll all the way to the bottom to change this value 😆 ". I agree that legibility should take priority over that though.